ivy_tests/test_ivy/helpers/hypothesis_helpers/array_helpers.py

```diff
@@ -1198,7 +1545,7 @@ def arrays_for_pooling(
         )
     if array_dim == 3:
         kernel = draw(st.tuples(st.integers(1, in_shape[1])))
-    new_kernel = kernel
+    new_kernel = kernel  # TODO: This will raise an error if array_dim not in [5, 4, 3]
```

Comment:
Hi @CatB1t, can you take a look at this?

Reply:
Hello @ZiadAmerr , it's better to reach out to the author of the code through git blame, meanwhile, we can merge it if it's ready as it doesn't seem to be a docs-related issue.